docs/book/src/tasks/experimental-features/cluster-class/write-clusterclass.md

```diff
@@ -676,6 +677,101 @@ individually.
 
 </aside>
 
+## Limitations of ClusterClass with patches
+
+JSON patches are limited by the specification [RFC6902] in not being able to add values to not-existing targets.
+
+The topology controller which applies the patches will return an error if a patch specified in 
+a ClusterClass matches all of the following conditions:
```

Comment:
```suggestion
As a consequence ClusterClass authors should pay special attention when the following conditions apply in order to prevent errors when a patch is applied:
```